src/index.ts

```diff
@@ -3,18 +3,63 @@ import rollCall from "rollCall";
 import { initIndexMappping } from "elasticsearch";
 import dotenv from "dotenv";
 import connectMongo from "connectMongo";
-import mongoose from "mongoose";
+import { Kafka } from "kafkajs";
+import * as swaggerUi from "swagger-ui-express";
+import path from "path";
+import yaml from "yamljs";
+
+import toProgramUpdateEvent from "toProgramUpdateEvent";
+import {
+  CLINICAL_PROGRAM_UPDATE_TOPIC,
+  KAFKA_CONSUMER_GROUP,
+  KAFKA_BROKERS,
+  PARTITIONS_CONSUMED_CONCURRENTLY,
+  PORT
+} from "config";
+import statusReport from "./statusReport";
+import logger from "logger";
 
 dotenv.config();
 
+const statusReporter = statusReport();
```

Comment:
you will need an express api for more than status reports 
I would expose a more generic api